src/lapack/laed4.cpp

```diff
@@ -0,0 +1,35 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2022, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#include <mkl.h>
```

Comment:
Does this mean MKL is required or do the other implementations expose this header as well?

Reply:
Unfortunately yes, MKL is required. I can invest some time to generalize if anyone is using a different LAPACK backend? Otherwise, I hope there is a new lapackpp release soon (it's been over a year since the last one) that will take care of that.

Reply:
Fine by me. I also only use MKL. But then it might be good to at least add a (temporary?) CMake check to disallow `DLAF_WITH_MKL=OFF`.

Reply:
Good point, I pushed a commit to do that : https://github.com/eth-cscs/DLA-Future/pull/550/commits/163018683bd3517ecd9ebc6b19580cd7e8703a32

Reply:
Can I suggest an easy approach that uses lapackpp utilities and works for all LAPACK supported by it?
https://github.com/eth-cscs/DLA-Future/commit/93d51bf9d5d55ad587dd99b3b10efbfcd8738e31